### ISSUE TITLE
Filter large database list in Postgresql

### ISSUE THREAD

**Issue by BinarySpike** [OmniDB/OmniDB#1240]
I'm running a heroku instance with a Postgres add-on.  The server has ~2000 databases.  I can't find a way to filter to only show my database, is there a feature in OmniDB for that?

**Comment by pgaro** [comment]
I don't think so. Note the same is true for a lot of schemas, or a lot of tables/functions/views or anything within a single schema really.

As I see it, even though intended differently, my feature idea #1233 would implicitly help with this issue as well. (At least is you define a separate connection for every database you actively care about.)

Regards
pgaro
